src/templates/createnewtask/controller.js

```diff
@@ -14,7 +14,11 @@
         // Files
         $scope.fileProvision = settings.fileProvisionTypes[0];
         $scope.files = {};
-
+        //Supported knowledge bases
```

Comment:
This is acceptable in some test, but not in code, which was denoted as solving #27. Discuss with @brodecva  how to manage reviews and when to close issues.